IOSExercises/Views/ProfileViewController.m

```diff
@@ -31,9 +34,27 @@ - (void)viewDidLoad {
     if (UIUserInterfaceIdiomPhone == [[UIDevice currentDevice] userInterfaceIdiom]) {
         [self activateIPhoneConstraints];
     }
+
+    UIDeviceOrientation orientation = self.view.bounds.size.width > self.view.bounds.size.height ? UIDeviceOrientationLandscapeRight: UIDeviceOrientationPortrait;
+
+    [self setupConstraints:orientation];
+}
+
+#pragma mark - Screen Navigation Methods
+
+- (IBAction)pushDetailsViewController:(UIButton *)sender {
+    if (!self.detailsViewController) {
+        self.detailsViewController = [[DetailsViewController alloc] initWithNibName:@"DetailsViewController"
+                                                                        bundle:[NSBundle mainBundle]];
+    }
+
+    self.detailsViewController.title = @"Hello, hello!"; // later dapat value ni sa profile view label
```

Comment:
Comments in English (project-standard, for our counterparts from other countries)